docs/guide/api-plugin.md

```diff
@@ -172,6 +172,8 @@ Note that the [`moduleParsed`](https://rollupjs.org/plugin-development/#modulepa
 
 Vite plugins can also provide hooks that serve Vite-specific purposes. These hooks are ignored by Rollup.
 
+Some Rollup hooks have the `options` parameter that are also extended with additional properties. You can read more in the [SSR documentation](/guide/ssr#ssr-specific-plugin-logic).
```

Comment:
I think maybe it is better to move this line to the previous section as it only affects rollup hooks. What if we add it to line 161 as
```
These Rollup hooks have an extended `options` parameter with additional Vite-specific properties. You can read more in the [SSR documentation](/guide/ssr#ssr-specific-plugin-logic).
```

Reply:
Good idea. I've moved them up in b759e92